 .code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,147 @@
+reset_searchable_attributes_1: |-
+get_attributes_for_faceting_1: |-
+update_attributes_for_faceting_1: |-
+reset_attributes_for_faceting_1: |-
+get_displayed_attributes_1: |-
+update_displayed_attributes_1: |-
+reset_displayed_attributes_1: |-
+get_accept_new_fields_1: |-
+update_accept_new_fields_1: |-
+get_index_stats_1: |-
+get_indexes_stats_1: |-
+get_health_1: |-
+update_health_1: |-
+get_version_1: |-
+get_pretty_sys_info_1: |-
+get_sys_info_1: |-
+distinct_attribute_guide_1: |-
+field_properties_guide_searchable_1: |-
+field_properties_guide_displayed_1: |-
+filtering_guide_1: |-
+filtering_guide_2: |-
+filtering_guide_3: |-
+filtering_guide_4: |-
+search_parameter_guide_query_1: |-
+search_parameter_guide_offset_1: |-
+search_parameter_guide_limit_1: |-
+search_parameter_guide_retrieve_1: |-
+search_parameter_guide_crop_1: |-
+search_parameter_guide_highlight_1: |-
+search_parameter_guide_filter_1: |-
+search_parameter_guide_filter_2: |-
+search_parameter_guide_matches_1: |-
+settings_guide_stop_words_1: |-
+settings_guide_ranking_rules_1: |-
+settings_guide_distinct_1: |-
+settings_guide_searchable_1: |-
+settings_guide_displayed_1: |-
+settings_guide_accept_new_fields_1: |-
+documents_guide_add_movie_1: |-
+search_guide_1: |-
+search_guide_2: |-
+getting_started_create_index_md: |-
+  ```bash
+    go get -u github.com/meilisearch/meilisearch-go
+  ```
+
+  ```go
+    import (
+        "github.com/meilisearch/meilisearch-go"
+    )
+  ```
+
+  ```go
+    var client = NewClient(meilisearch/Config{
+        Host: "http://127.0.0.1:7700",
+    })
+
+    createIndexResp, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
+        UID: "movies",
+    })
+
+    if err != nil {
+        panic(err)
+    }
+  ```
+
+  [About this package](https://github.com/meilisearch/meilisearch-go/)
+getting_started_add_documents_md: |-
+  ```go
+    import (
+        "os"
+        "encoding/json"
+        "io/ioutil"
+        "github.com/meilisearch/meilisearch-go"
+    )
+  ```
+
+  ```go
+    jsonFile, _ := os.Open("movies.json")
+    defer jsonFile.Close()
+
+    byteValue, _ := ioutil.ReadAll(jsonFile)
+    var movies []map[string]interface{}
+    json.Unmarshal(byteValue, &movies)
+
+    updateDocumentsResp, err := client.Documents("movies").AddOrUpdate(movies)
+
+    if err != nil {
+        panic(err)
+    }
+  ```
+  [About this package](https://github.com/meilisearch/meilisearch-go/)
+getting_started_search_md: |-
+  ```go
+    results, err := client.Search("movies").Search(meilisearch.SearchRequest{
+        Query: "botman",
+    })
+    if err != nil {
+        panic(err)
+    }
+  ```
+
+  [About this package](https://github.com/meilisearch/meilisearch-go/)
+faceted_search_update_settings_1: |-
+faceted_search_facet_filters_1: |-
+faceted_search_facets_distribution_1: |-
+faceted_search_walkthrough_attributes_for_faceting_1: |-
+faceted_search_walkthrough_facet_filters_1: |-
+faceted_search_walkthrough_facets_distribution_1: |-
```

Comment:
Add new line at the end of file. Use the plugin editorconfig to your Text Editor to avoid this error ;)

Reply:
Hum I realize there is no editorconfig file in the go package 😂

Reply:
Exactly, I just created it ;)